src/MongoDB.Analyzer.Helpers/Builders/MqlGenerator.cs

```diff
@@ -22,6 +22,45 @@
 
 namespace MongoDB.Analyzer.Helpers.Builders
 {
+    internal static class Renderer
```

Comment:
Extract to it's own file.

Reply:
i don't think i can move the Renderer class to another file. Reason being is that the Render method won't be in the syntax Tree of the MqlGenerator class
so when we invoke Get_MQL()
it won't know how to resolve Renderer.Render()
my theory was confirmed because I actually tried moving it to another file and it destroyed all the test cases